vault/dendron.topic.publish.cook.common.md

```diff
@@ -9,7 +9,7 @@ created: 1631132278348
 ### Setup Dendron CLI
 ```
 npm init -y 
-npm install --save @dendronhq/dendron-cli
+npm i @dendronhq/dendron-cli
```

Comment:
can we keep this as `npm install --save`?
- `install` is better than `i` just to clear up ambiguity
- `--save` is for including this is `package.json`

Reply:
As of npm@5.0.0 (May, 2017) `--save` is deprecated.
> `--save` is no longer necessary. All installs will be saved by default. You can prevent saving with `--no-save`. Installing optional and dev deps is unchanged: use `-D`/`--save-dev` and `-O`/`--save-optional` if you want them saved into those fields instead. 

Source: https://blog.npmjs.org/post/161081169345/v500

Reply:
as for using `install` over `i`, I've made that change now and it should be ready for review again